src/Piece/PieceForm/PieceForm.test.js

```diff
@@ -405,5 +408,23 @@ describe('PieceForm', () => {
         expect.anything(),
       );
     });
+
+    it('should `Delete` and `Unreceive` buttons be disabled if the piece is bound', async () => {
+      rerenderComponent(
```

Comment:
Why do you use such an approach instead of `renderPieceForm` with necessary props?

Reply:
Good catch

Reply:
updated